mozjs/src/conversions.rs

```diff
@@ -525,6 +526,16 @@ impl FromJSValConvertible for f64 {
     }
 }
 
+/// Copies chars to the string
+unsafe fn fast_copy(chars: &[u8]) -> String {
+    let mut v = Vec::with_capacity(chars.len() * 2);
+    v.set_len(chars.len() * 2);
```

Comment:
This is undefined behavior. See Safety requirements of [set_len](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.set_len)